src/packing.cc

```diff
@@ -1676,6 +1677,64 @@ void xnn_pack_kai_qs4_weights_and_biases(
         &kai_params);
   }
 }
+
+size_t xnn_packed_stride_kai_qb4_weights_and_biases(
+    const struct xnn_gemm_config* gemm_config, size_t k, size_t block_size,
+    size_t extra_bytes) {
+  const uint32_t kr = UINT32_C(1) << gemm_config->log2_kr;
+
+  const size_t kai_num_bytes_sum_rhs = sizeof(float);
+  const size_t kai_num_bytes_bias = sizeof(float);
+  // perhaps derive Bf16 from gemm-config?
+  // This needs to be updated in the kleidi branch to be in header
+  // return kai_rhs_packed_stride(k, /*nr=*/1, kr, block_size, Bf16);
+  const size_t num_bytes_multiplier_rhs = sizeof(uint16_t);
+  const size_t num_blocks_per_row = k/block_size;
+  const size_t num_bytes_per_block = (block_size / 2) + num_bytes_multiplier_rhs;
+  return 1 * ((num_bytes_per_block * num_blocks_per_row) + kai_num_bytes_sum_rhs + kai_num_bytes_bias);
```

Comment:
Is there no corresponding `kai_get_rhs_packed_stride_rhs_pack_kxn_*` that this should call instead of doing the computation itself?

The problem is that this code assumes that the KleidiAI implementation will never change.